src/Domain/User/User.php

```diff
@@ -135,4 +135,9 @@ public function updateProfile(
         $this->displayMyAge = $displayMyAge;
         $this->birthday = $birthday;
     }
+
+    public function isCompleted(): bool
```

Comment:
(Détail) En anglais to complete (v) = compléter au sens de "terminer", mais complete (ajd) = complet / total / parfait / exhaustive). Donc j'utiliserais plutôt l'adjectif "complete"

```suggestion
    public function isComplete(): bool
```